app/services/weets.js

```diff
@@ -0,0 +1,17 @@
+const axios = require('axios').default;
+
+const logger = require('../logger');
+const config = require('../../config').common.apiGeekJoke;
+
+exports.getRandomJoke = async () => {
+  try {
+    const response = await axios({
+      url: `${config.url}/api?format=json`,
+      method: 'GET'
+    });
+    return response.data;
+  } catch (error) {
+    logger.error(error.message);
+    return error.message;
```

Comment:
I would expect something like
throw errors.external_api_error ('Error getting random joke');

making use of the project's error handler